dotnetv3/S3/README.md

```diff
@@ -1,4 +1,4 @@
-# Amazon S3 code examples for .NET
+git# Amazon S3 code examples for .NET
```

Comment:
Looks like you typed into the wrong window. Remove `git`

Reply:
fixed